homeassistant/components/huawei_lte/strings.json

```diff
@@ -85,8 +85,8 @@
       "description": "Suspends integration. Suspending logs the integration out from the router, and stops accessing it. Useful e.g. if accessing the router web interface from another source such as a web browser is temporarily required. Invoke the resume_integration service to resume.\n.",
       "fields": {
         "url": {
-          "name": "URL",
-          "description": "URL of router to resume integration for; optional when only one is configured."
+          "name": "[%key:common::config_flow::data::url%]",
+          "description": "[%key:component::huawei_lte::services::resume_integration::fields::url::description%]"
```

Comment:
This being a duplicate is actually a bug, it should read "...to suspend integration for; ...". Fix in #96450